test/SILOptimizer/outliner.swift

```diff
@@ -175,3 +175,6 @@ extension Operation {
   }
 }
 
+public func dontCrash(a: Any)  {
```

Comment:
Can you pattern match the SIL to show that there's actually an opened type in there, or even write a SIL test case?